src/Admin/src/RoutesDelegator.php

```diff
@@ -16,12 +28,33 @@ public function __invoke(ContainerInterface $container, string $serviceName, cal
         /** @var Application $app */
         $app = $callback();
 
-        $app->route(
-            '/admin[/{action}[/{uuid}]]',
-            AdminController::class,
-            [RequestMethodInterface::METHOD_GET, RequestMethodInterface::METHOD_POST],
-            'admin'
-        );
+        $app->get('/admin/add', GetAdminFormHandler::class, 'admin::get-admin-form');
+        $app->post('/admin/add', CreateAdminHandler::class, 'admin::create-admin');
+
+        $app->get('/admin/edit/{uuid}', GetEditAdminFormHandler::class, 'admin::get-edit-admin-form');
+        $app->post('/admin/edit/{uuid}', EditAdminHandler::class, 'admin::edit-admin');
+
+        $app->get('/admin/delete/{uuid}', GetDeleteAdminFormHandler::class, 'admin::get-delete-admin-form');
+        $app->post('/admin/delete/{uuid}', DeleteAdminHandler::class, 'admin::delete-admin');
+
+        $app->get('/admin/list', AdminListHandler::class, 'admin::list');
```

Comment:
```suggestion
        $app->get('/admin/list-admin', ListAdminHandler::class, 'admin::list-admin');
```
1. Rename `AdminListHandler` to `ListAdminHandler`.
Why?
For consistecy across the handlers (**Create**AdminHandler, **Edit**AdminHandler, **Delete**AdminHandler, ...) - first, the action, then the subject. When quickly looking through the handlers in a directory, it is more relevant to start the file names with the action they perform.
**Example**: you are looking for the handler which lists the admin accounts, your eyes search for `List`, then `Admin`.

2. Since the `Admin` module lists more than one type of resources (admin accounts, login attempts), `list-*` routes should clearly specify what they are listing.